base/server/python/pki/server/cli/cert.py

```
@@ -393,7 +393,7 @@ def execute(self, argv, args=None):
 
         if not subsystem:
             raise Exception(
-                'No %s subsystem in instance %s' % (subsystem.type, instance_name))
+                'No {} subsystem in instance {}'.format(subsystem.type, instance_name))
```

Ruff (0.15.2)

[warning] Create your own exception

(TRY002)